ui/pages/confirmations/send/send.test.js

```diff
@@ -161,6 +161,11 @@ const baseStore = {
   },
 };
 
+const placeholderText =
+  process.env.METAMASK_BUILD_TYPE === 'flask'
```

Comment:
I still don't understand this. The build type is never `flask` when running unit tests?

Reply:
Ah forgot to clean this up with the other changes I had made, fixed!